com.unity.shadergraph/Editor/Data/Graphs/Texture2DArrayShaderProperty.cs

```diff
@@ -7,6 +7,39 @@ namespace UnityEditor.ShaderGraph.Internal
     [FormerName("UnityEditor.ShaderGraph.Texture2DArrayShaderProperty")]
     public sealed class Texture2DArrayShaderProperty : AbstractShaderProperty<SerializableTextureArray>
     {
+        public static readonly Texture2DArrayShaderProperty kLightmapsArray = new Texture2DArrayShaderProperty()
```

Comment:
I don't love having these definitions in this class, especially if they are public.  They would be better in a separate file for unity lighting definitions.

Reply:
moved to new file